src/flow/ext_multiroute_flow.jl

```diff
@@ -181,7 +181,7 @@ function slope{T<:AbstractFloat}(
   )
   slope = 0
   for e in edges(flow_graph)
-    if cut[dst(e)] - cut[src(e)] > 0 &&
+    if cut[dst(e)] == 2 > cut[src(e)] &&
```

Comment:
Huh - I've seen chaining with `>` and `<` but never with `==`. That's pretty neat, though it's not immediately intuitive. Happy to keep it in there, though. Would you mind putting in a comment explaining the chain so that when I look at it in 6 months I remember what it's supposed to do? :)